packages/ui/src/helpers/authenticator/utils.ts

```diff
@@ -32,6 +33,27 @@ export const censorPhoneNumber = (val: string): string => {
   return split.join('');
 };
 
+export const defaultAuthHubHandler: HubHandler = (data, service) => {
```

Comment:
What do you think about adding unit testing for this utility? Just thinking that it could be useful since we don't cover this with integ testing (afaik)

Reply:
I like it. There is however an integration test for these though at `/hub-events` ([.feature file](https://github.com/aws-amplify/amplify-ui/blob/main/packages/e2e/features/ui/components/authenticator/hub-events.feature)), which mocks and tests hub events and see if authenticator reacts accordingly.

That said though, it only tests for happy cases (as is with other e2e tests), so I do see value in adding unit tests. But currently `ui` packages do not have unit tests set up, so I'm thinking we want to scope this out to a bigger PR that adds low-level testing on UI repo. What do you think? For now, integration tests at least covers all the known auth hub events.

Reply:
That works for me 😄  

If we don't already, can we add a backlog ticket for setting up unit tests where needed?

Reply:
Done! https://app.asana.com/0/1201736086077862/1202277146130103/f